lib/extensions/firestore_extensions.dart

```diff
@@ -132,11 +150,15 @@ extension ConnectAndConvert on FirebaseFirestore {
         controller.add(UpdateProfilePage(
             userId: leaguer.uid, leaguerPhotoURL: leaguer.photoURL));
       } catch (error, trace) {
-        controller.add(AddProblem.from(
-          message: error.toString(),
-          type: ProblemType.observeProfile,
-          traceString: trace.toString(),
-        ));
+        controller.add(
```

Comment:
trev omg, this stuffs crazy. cant believe what world youve been in. good job 🙏 🙏